taxi/views.py

```diff
@@ -17,3 +18,46 @@ def index(request):
     }
 
     return render(request, "taxi/index.html", context=context)
+
+
+class ManufacturerListView(generic.ListView):
+    # queryset = Manufacturer.objects.order_by("name")
+    model = Manufacturer
+    ordering = ["name"]
```

Comment:
replace ordering by your commented attribute

```suggestion
    queryset = Manufacturer.objects.order_by("name")
```